content/en/docs/installing-on-linux/introduction/storage-configuration.md

```diff
@@ -1,20 +1,19 @@
 ---
 title: "Persistent Storage Configuration"
-keywords: 'kubernetes, docker, kubesphere, storage, volume, PVC'
+keywords: 'Kubernetes, docker, KubeSphere, storage, volume, PVC'
 description: 'Persistent Storage Configuration'
 
 linkTitle: "Persistent Storage Configuration"
 weight: 2140
 ---
 ## Overview
-Persistence volume is **Must** for Kubesphere. So before installation of Kubesphere,  one **default** 
-[StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) and corresponding storage plugin should be installed on the Kubernetes cluster.
-As different users may choose different storage plugin, [KubeKey](https://github.com/kubesphere/kubekey) supports to install storage plugin by the way of 
-[add-on](https://github.com/kubesphere/kubekey/blob/v1.0.0/docs/addons.md). This passage will introduce add-on configuration for some mainly used storage plugin.
+A persistent volume is a **Must** for KubeSphere. Therefore, before the installation of KubeSphere, a **default** 
```

Comment:
`Persistent volumes are` or `Persistent storage service`

Reply:
"Persistent volumes are" is also correct. "A persistent volume is" is correct as well, as I find the description in the official website of Kubernetes:
![image](https://user-images.githubusercontent.com/65327072/92347093-70537380-f101-11ea-98d0-6ab934846caf.png)

Reply:
Does `A persistent volume` mislead the user that the installation of KubeSphere requires only one PV?

Reply:
sure, changed